repo_id,repo_full_name,file,file_sha1,match
313855291,lindemer/CVE-2019-11043,exploit.py,50d4d1bc2acbab2e1e9bc2c07020d602f7fac289,CVE-2019-11043
313855291,lindemer/CVE-2019-11043,README.md,9f8d0c21950dc7280e50fca5e631af20e0e1280c,CVE-2019-11043
313855291,lindemer/CVE-2019-11043,_GITHUB_REPO_METADATA_,,CVE-2019-11043
